ec2.tf

```diff
@@ -0,0 +1,109 @@
+# Create a security group
+resource "aws_security_group" "loaf_sg_ec2" {
+  name        = "loaf_sg_ec2"
+  description = "Security group for EC2"
+
+  ingress {
+    from_port   = 443
+    to_port     = 443
+    protocol    = "tcp"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+
+  ingress {
+    from_port   = 22
+    to_port     = 22
+    protocol    = "tcp"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+
+  ingress {
+    from_port   = 80
+    to_port     = 80
+    protocol    = "tcp"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+
+  egress {
+    from_port   = 0
+    to_port     = 0
+    protocol    = "-1"
+    cidr_blocks = ["0.0.0.0/0"]
+  }
+}
+
+
+resource "aws_instance" "loaf_app" {
+  ami                    = "ami-0f8b8f874036055b1"
+  instance_type          = "t2.micro"
+  vpc_security_group_ids = [aws_security_group.loaf_sg_ec2.id]
+
+  user_data = <<-EOF
+    #!/bin/bash
+
+    # This is a sample shell script
+    cd ./home/ubuntu
+    sudo apt update
+    sudo apt install -y apt-transport-https ca-certificates curl software-properties-common
+    curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo apt-key add -
+    sudo add-apt-repository "deb [arch=amd64] https://download.docker.com/linux/ubuntu focal stable"
+    sudo apt update
+    sudo apt install -y docker-ce
+    git clone https://github.com/Capstone2401/backend-app.git
+    sudo systemctl start docker
+    cd backend-app
+    echo "REDSHIFT_CONN_STRING=postgresql://${aws_redshift_cluster.redshift_cluster.master_username}:${aws_redshift_cluster.redshift_cluster.master_password}@${aws_redshift_cluster.redshift_cluster.endpoint}/${aws_redshift_cluster.redshift_cluster.database_name}" > .env
+    sudo docker build -t backend-app:loaf .
+    sudo apt install docker-compose
+    sudo docker compose --env-file .env up -d
+
+  EOF
+
+  tags = {
+    Name = "loaf_application"
+  }
+}
+
+/* resource "aws_acm_certificate" "cert" {
```

Comment:
Can you remove the commented out code before we merge?

Reply:
The above code has been deleted out of the file.